src/test/java/com/amazon/opendistroforelasticsearch/security/ssl/SSLTest.java

```diff
@@ -633,7 +633,7 @@ public void testNodeClientSSL() throws Exception {
 
     @Test
     public void testTransportClientSSLFail() throws Exception {
-        thrown.expect(NoNodeAvailableException.class);
+        thrown.expect(IllegalStateException.class);
```

Comment:
Earlier we were allowing transport client creation with an empty truststore file. I added a stronger validation for truststore validation so right now we will prevent transport client and OpenDistroPlugin will not load, hence cause this `IllegalStateException`